sdk/ai/azure-ai-projects/azure/ai/projects/operations/_operations.py

```diff
@@ -1576,7 +1576,7 @@ def delete(self, name: str, version: str, **kwargs: Any) -> None:  # pylint: dis
 
         response = pipeline_response.http_response
 
-        if response.status_code not in [204, 200]:
+        if response.status_code not in [204]:
             map_error(status_code=response.status_code, response=response, error_map=error_map)
```

Comment:
Please revert this change. This is a manual fix to auto-generated code, to temporary overcome a service bug. Just remove this file from your PR.